components/verify/Verifier.tsx

```diff
@@ -66,16 +66,33 @@ const Verifier: React.FC<VerifierProps> = ({ wrappedDocument }) => {
 
       setFragmentVerificationStatus({ DOCUMENT_STATUS, DOCUMENT_INTEGRITY, ISSUER_IDENTITY });
 
+      const isRevoked = verifyUtils
+        .getDocumentStatusFragments(fragments)
+        .filter((fragment) => verifyUtils.isInvalidFragment(fragment))
+        .some((invalidFragment: any) => invalidFragment.data.revokedOnAny);
```

Comment:
Let's add a comment here so that we can update this in the future:

```suggestion
        .some((invalidFragment: any) => invalidFragment.data.revokedOnAny); // FIXME: To update oa-verify library so that types are automatically inferred
```